addons/mail/static/src/web/discuss_app/sidebar.xml

```diff
@@ -82,17 +82,17 @@
 
 <t t-name="mail.DiscussCategoryItem" owl="1">
     <t t-set="counter" t-value="threadService.getCounter(thread)"/>
-    <button class="o-mail-DiscussCategoryItem o-mail-DiscussSidebar-item btn btn-secondary d-flex align-items-center w-100 px-0 py-2 border-0 text-reset"
+    <button class="o-mail-DiscussCategoryItem o-mail-DiscussSidebar-item btn btn-secondary d-flex align-items-center w-100 px-0 py-2 border-0 text-reset rounded-0"
         t-att-class="{
             'o-bg-inherit': threadLocalId !== store.discuss.threadLocalId,
-            'o-active bg-300': threadLocalId === store.discuss.threadLocalId,
+            'o-active bg-200': threadLocalId === store.discuss.threadLocalId,
```

Comment:
Do we want the hovering and active states to have the same color?

Reply:
It's not a big deal to have difference between active and hover.
Here I make same visual because this avoids going to darker colors (`bg-300`), which are too dark with new coloring with MILK.

Reply:
For me I would say that it's great to have a different hover and active, otherwise it's hard to know which is which.